File: programers_Lv1/나누어 떨어지는 숫자 배열.py
# https://school.programmers.co.kr/learn/courses/30/lessons/12910?language=python3
def solution(arr, divisor):
    return sorted([n for n in arr if n%divisor == 0]) if len([n for n in arr if n%divisor == 0]) > 0 else [-1]

print(solution([5, 9, 7, 10], 5))

File: programers_Lv1/나누어 떨어지는 숫자 배열.js
// https://school.programmers.co.kr/learn/courses/30/lessons/12910?language=javascript
function solution(arr, divisor) {
  var answer = arr.filter((el) => el % divisor === 0).sort((a, b) => a - b);
  return answer.length === 0 ? [-1] : answer;
}

console.log(solution([5, 9, 7, 10], 5));
